include/ltest-macros.lfe

```diff
@@ -87,6 +87,35 @@ A simple wrapper macro for defining the tear-down function in a fixture."
 ;;; Test case macros
 ;;;===================================================================
 
+(eval-when-compile
+  (defun is-named-tuple (t)
+    (progn ;(lfe_io:format "~p" (list t)) ;for debugging
```

Comment:
Yeah, I left it to make it easy to uncomment the debugging code, but I'll get rid of it